protected/humhub/modules/space/widgets/Menu.php

```diff
@@ -46,15 +47,31 @@ public function init()
 
         $this->panelTitle = Yii::t('SpaceModule.base', '<strong>Space</strong> menu');
 
+        parent::init();
+
+        if (!$this->space->isMember() && $this->space->visibility == Space::VISIBILITY_NONE)
```

Comment:
Please always use brackets also for one line statements

Reply:
Also not really sure whats exactly happening here. Could you describe the expected behavior here? Should the Stream be visible in this case? Perhaps an inline doc with some further infos would be nice.

Reply:
It was only `$this->space->isMember()` before, to hide any menu items except about and stream for non-member users.  @luke- wanted to add `&& $this->space->visibility == Space::VISIBILITY_NONE` which make other items hide for non-members only if a space has `VISIBILITY_NONE`.